### PR TITLE
Updated Squashtoberfest for 2019

### LINKED ISSUE #360: Squashtoberfest 2019 (Magento / Adobe)
## Wild swag opportunity appeared!

- [x] I've checked that this isn't a duplicate issue.

Q. Which company offers it?  
A. : Magento / Adobe

Q. Where have you seen the announcement (URL)?  
A. : https://opensource.adobe.com/squashtoberfest/

Q. What kind of swag (clothing, stickers, ...)?  
A. : clothing

Q. Have you received swag from this?  
A. : no

Q. How long did it take?  
A. : N/A

Q. Does it ship worldwide?  
A. : Unsure

- [x] I've checked that the promotion is legit.
- [x] I've checked that it's still available to the public.

### PR DESCRIPTION
**Pull request by gpjacobs**
Fixes #360

- [x] I've checked that this isn't a new swag opportunity proposal.
- [x] I've checked that this isn't a duplicate pull request.

Updates the Squashtoberfest entry from last year to remove the expired tag, and update the details, link, and image.

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -173,11 +173,11 @@
   {
     "name": "Magento",
```
Comment (aslafy-z) [inline data.json]: Can you change for either Magento + Adobe or the other way around?

Comment (gpjacobs) [reply data.json]: Updated accordingly

File: data.json [hunk 2]
```diff
@@ -171,13 +171,13 @@
...
+    "name": "Magento + Adobe",
     "difficulty": "hard",
-    "description": "Have 5 merged PRs or validate at least 10 issues in October 2018, get a Squashtoberfest T-Shirt!",
-    "reference": "https://github.com/magento/magento2/wiki/SQUASHTOBERFEST-2018",
-    "image": "https://raw.githubusercontent.com/wiki/magento-engcom/msi/images/squash2018.png",
-    "dateAdded": "2018-10-23T10:17:39.000Z",
-    "tags": ["clothing", "hacktoberfest", "expired"]
+    "description": "Submit 5 or more pull requests during October 2019 to any open repositories under github.com/adobe or github.com/magento, get a Squashtoberfest T-Shirt!",
+    "reference": "https://opensource.adobe.com/squashtoberfest/",
+    "image": "https://jvdeh29369.i.lithium.com/t5/image/serverpage/image-id/14947i2CC7C22EFCA61CAB?v=1.0",
```
Comment (aslafy-z) [inline data.json]:
```suggestion
    "image": "https://i.imgur.com/yt3iUBB.png",
```

File: data.json [hunk 3]
```diff
@@ -171,13 +171,13 @@
...
   {
-    "name": "Magento",
+    "name": "Magento + Adobe",
     "difficulty": "hard",
-    "description": "Have 5 merged PRs or validate at least 10 issues in October 2018, get a Squashtoberfest T-Shirt!",
-    "reference": "https://github.com/magento/magento2/wiki/SQUASHTOBERFEST-2018",
-    "image": "https://raw.githubusercontent.com/wiki/magento-engcom/msi/images/squash2018.png",
-    "dateAdded": "2018-10-23T10:17:39.000Z",
-    "tags": ["clothing", "hacktoberfest", "expired"]
+    "description": "Submit 5 or more pull requests during October 2019 to any open repositories under github.com/adobe or github.com/magento, get a Squashtoberfest T-Shirt!",
```
Comment (aslafy-z) [inline data.json]:
```suggestion
    "description": "Submit 5 or more pull requests during October 2019 to any open repositories under Adobe or Magento organizations on GitHub, get a Squashtoberfest T-Shirt!",
```

**Comment by aslafy-z** [comment]
@vikaspotluri merge when you are ready ;)

**Pull request merged**